syntax/closesection.php

```diff
@@ -0,0 +1,35 @@
+<?php
+if(!defined('DOKU_INC')) die();
+
+if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
+require_once(DOKU_PLUGIN.'syntax.php');
```

Comment:
these three lines can be removed (PSR-2)

Reply:
fixed